app/deepseg.cc

```diff
@@ -389,7 +389,9 @@ int main(int argc, char* argv[]) try {
 		fprintf(stderr, "-h            Specify the video stream height\n");
 		fprintf(stderr, "-f            Specify the camera video format, i.e. MJPG or 47504A4D.\n");
 		fprintf(stderr, "-t            Specify the number of threads used for processing\n");
-		fprintf(stderr, "-b            Specify the background image\n");
+		fprintf(stderr, "-b            Specify the background (any local or network OpenCV source) e.g.\n");
+		fprintf(stderr, "                images/total_landscaping.jpg,\n");
+		fprintf(stderr, "                https://github.com/floe/backscrub/raw/experimental/images/rotating_earth.webm\n");
```

Comment:
```suggestion
		fprintf(stderr, "-b            Specify the background (any local or network OpenCV source) e.g.\n");
		fprintf(stderr, "                local: images/total_landscaping.jpg,\n");
		fprintf(stderr, "                net: 
 https://upload.wikimedia.org/wikipedia/commons/transcoded/9/99/Animation_of_Rotating_Earth_at_Night.webm/Animation_of_Rotating_Earth_at_Night.webm.360p.vp9.webm\n");
```

Maybe look for a shorter link for the network example …

Reply:
Looks like Github have a link shortener we can use, so:
https://github.com/floe/backscrub/raw/experimental/images/rotating_earth.webm maps to
https://git.io/JE9o5
I'll pop that in.

Reply:
For an animated GIF, Wikipedia have a reasonable example, so using that.
https://upload.wikimedia.org/wikipedia/commons/d/d3/Newtons_cradle_animation_book_2.gif